### ISSUE TITLE
[FEATURE] Remove warning message from the CM Bot when a user updates their message.

### ISSUE THREAD

**Issue by unnikrishnan2002** [commclassroom/classroom-monitor-bot#150]
### Is your feature request related to a problem? Please describe.

The Bot gives warning when inclusive language is not used. But the warning message is not removed after the user updates their message accordingly.

### Describe the solution you'd like

The Bot should remove the warning message after the user updates their message

**Comment by genzyy** [comment]
Hey, I have an idea to implement this. Let me know if I can work on it or is there a better way to do this,

```javascript

const csgo_scam =
  "hello i leave from csgo and give all my inventory, the first 3 will get trade";

const csgo_res = csgo_scam.match(/csgo|trade|inventory/g);

console.log(csgo_res);

const ban_requirements = {
  csgo: 3, // [csgo, inventory, trade]
  discord1: 2, // [discord, nitro]
  discord2: 3, // [discord, nitro, steam]
  discord3: 3, // [discord, nitro, month]
  discord4: 3, // [referral, discord, nitro]
};

if (csgo_res.length === ban_requirements.csgo) {
  // message.delete();
  // or
  // ban(user);
  console.log("delete message");
}

// same for other scams ...

```

- Line 2 matches given string with the given words using regex and returns all the words that matched.
- So the returned array's length is same as number of words we matched, then its a scam message.

**Comment by Panquesito7** [comment]
> Hey, I have an idea to implement this. Let me know if I can work on it or is there a better way to do this,
> 
> ```js
> const csgo_scam =
>   "hello i leave from csgo and give all my inventory, the first 3 will get trade";
> 
> const csgo_res = csgo_scam.match(/csgo|trade|inventory/g);
> 
> console.log(csgo_res);
> 
> const ban_requirements = {
>   csgo: 3, // [csgo, inventory, trade]
>   discord1: 2, // [discord, nitro]
>   discord2: 3, // [discord, nitro, steam]
>   discord3: 3, // [discord, nitro, month]
>   discord4: 3, // [referral, discord, nitro]
> };
> 
> if (csgo_res.length === ban_requirements.csgo) {
>   // message.delete();
>   // or
>   // ban(user);
>   console.log("delete message");
> }
> 
> // same for other scams ...
> ```
> 
> * Line 2 matches given string with the given words using regex and returns all the words that matched.
> * So the returned array's length is same as number of words we matched, then its a scam message.

Looks good! 🚀 What do you think, @kaiwalyakoparkar, @SiddhantKhisty?

**Comment by SiddhantKhisty** [comment]
> > Hey, I have an idea to implement this. Let me know if I can work on it or is there a better way to do this,
> > ```js
> > const csgo_scam =
> >   "hello i leave from csgo and give all my inventory, the first 3 will get trade";
> > 
> > const csgo_res = csgo_scam.match(/csgo|trade|inventory/g);
> > 
> > console.log(csgo_res);
> > 
> > const ban_requirements = {
> >   csgo: 3, // [csgo, inventory, trade]
> >   discord1: 2, // [discord, nitro]
> >   discord2: 3, // [discord, nitro, steam]
> >   discord3: 3, // [discord, nitro, month]
> >   discord4: 3, // [referral, discord, nitro]
> > };
> > 
> > if (csgo_res.length === ban_requirements.csgo) {
> >   // message.delete();
> >   // or
> >   // ban(user);
> >   console.log("delete message");
> > }
> > 
> > // same for other scams ...
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * Line 2 matches given string with the given words using regex and returns all the words that matched.
> > * So the returned array's length is same as number of words we matched, then its a scam message.
> 
> Looks good! 🚀 What do you think, @kaiwalyakoparkar, @SiddhantKhisty?

@Panquesito7  correct me if I'm wrong, but this looks like a solution for a different issue entirely🙂
And we have solved the issue of scam links using becca bot

**Comment by kaiwalyakoparkar** [comment]
Yeah @SiddhantKhisty I feel the same. I don't see any code which checks for updated message or similar. I might be wrong. @genzyy could you please explain it maybe?

**Comment by genzyy** [comment]
My bad!, I mistakenly added this code on a different issue than which I intended this solution for. Really sorry for that.
